test/visibility/only/inheritance-subclassListed.good

```diff
@@ -0,0 +1,3 @@
+inheritance-subclassListed.chpl:30: In function 'main':
+inheritance-subclassListed.chpl:32: error: unresolved call 'owned Child.tertiaryMethod()'
+inheritance-subclassListed.chpl:32: note: because no functions named tertiaryMethod found in scope
```

Comment:
Should this also be more like

` inheritance-subclassListed2a.chpl:34: error: Cannot find symbol 'Child' in module 'definesTertiary'`

?